lib/LaTeXML/resources/RelaxNG/LaTeXML-misc.rnc

```diff
@@ -111,7 +111,7 @@ grammar {
    common.attrs.aria.implicit.link = empty
    SVG.Core.extra.attrib &= parent svgForeign.attributes
   include "svg11.rnc"
-  { SVG.foreignObject.content = parent Flow.model
+  { SVG.foreignObject.content = parent Inline.model
```

Comment:
Thinking out loud: Right, so the idea for switching to Inline.model is to leverage `ltx:inline-block` (or similar) auto-opening.

I think that would benefit from a schema comment - since the HTML+SVG spec and TeX would both permit pretty much any layout-oriented content in those nodes (e.g. a table, a citation, a math formula, but not a section or a bibliography). I think it would be helpful for the schema to document that, and mention the `inline-block` mechanism as the intended carrier for the effect.

I guess the alternative was to start as block-level and auto-open nodes that can hold Inline content?

Reply:
Certain TeX does, which is the issue; HTML sorta does, but when you start getting `p` in peculiar places, it has undesirable effects that can be papered over with CSS, but it gets complex & error-prone. So it's basically the same rationale for changing `td` content model to Inline.

In this recent set of changes, I'm finding `Flow` model kinda annoying for that reason. It probably contributes to the confusion dealing with figure & graphic layout since you're never quite sure what you're getting.